apps/post-producer/main.go

```diff
@@ -0,0 +1,176 @@
+package main
```

Comment:
Naming suggestion for service: `post-api` - just to make it clear we split the former `api` directory

Reply:
Will do in a separate PR.